frontend/src/views/CellGuide/components/CellGuideCard/index.tsx

```diff
@@ -166,8 +167,11 @@ export default function CellGuideCard({
   const handleChangeOrgan = (option: DefaultDropdownMenuOption | null) => {
     if (!option) return;
     setSelectedOrgan(option);
-    setSelectedOrganId(organsMap.get(option.name) ?? "");
-    track(EVENTS.CG_SELECT_TISSUE, { tissue: option.name });
+    const optionName =
+      option.name === TISSUE_AGNOSTIC ? ALL_TISSUES : option.name;
+    setSelectedOrganId(organsMap.get(optionName) ?? "");
+    // Continue tracking the analytics event as All Tissues
+    track(EVENTS.CG_SELECT_TISSUE, { tissue: optionName });
```

Comment:
i'm assuming it's better to continue tracking the analytics events as ALL_TISSUES